### PR TITLE
Modified index.css and index.less

### PR DESCRIPTION
**Pull request by jadeli1720**

### CODE REVIEW

File: less/index.less [hunk 1]
```diff
@@ -57,3 +57,386 @@ html, body {
...
+	margin: 0 0  40px;
+	padding: 30px 80px;
+	.shadow();
+	position: fixed;
+	top: 0%;
+	height: 100px;
+	width: 100vw;
+	z-index: 1;
+
+	.nav-list{
```
Comment (dpayton23) [inline less/index.less]: bp

Comment (dpayton23) [inline less/index.less]: bp

File: less/index.less [hunk 2]
```diff
@@ -57,3 +57,386 @@ html, body {
...
+		}
+	}
+}
+
+// About Section
+
+.about-row{
+	.custom-layout(space-between, flex-start);
+	
+	.about-column {
```
Comment (dpayton23) [inline less/index.less]: bp

File: less/index.less [hunk 3]
```diff
@@ -57,3 +57,386 @@ html, body {
...
+			.custom-layout(space-between, flex-start);
+			color: @primary;
+			margin-bottom:15px;
+
+			p{
+				font-size: 2.4rem;
+			}
+		}
+
+		.progress{
```
Comment (dpayton23) [inline less/index.less]: bp

File: less/index.less [hunk 4]
```diff
@@ -57,3 +57,386 @@ html, body {
...
+
+
+
+// Portfolio
+
+.portfolio-row {
+	display:flex;
+	justify-content: space-around;
+
+	.portfolio-cards {
```
Comment (dpayton23) [inline less/index.less]: bp

**Comment by jadeli1720** [comment]
Need changes to be merged to master. Have made further changes.

**Pull request merged**